containers/nginx-storage/delete-old-files.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+DIRECTORIES="$(echo /srv/storage/blender/{result/,'source/',verifier-output/})"
```

Comment:
is it good idea to replace it with a list, and then run `find` for each element in a list ? This is hacky, but less readable. Also, [useless use of echo](http://porkmail.org/era/unix/award.html) ;)

Reply:
Done.